ports/esp32s2/boards/lilygo_ttgo_t8_esp32s2/mpconfigboard.mk

```diff
@@ -0,0 +1,18 @@
+USB_VID = 0x239A
+USB_PID = 0x80AC
```

Comment:
please ask lilygo for a VID/PID instead of using unexpected makers'

Reply:
@ladyada `0x239A` is [Adafruits's vendor ID](https://devicehunt.com/view/type/usb/vendor/239A). 
I used it because many other boards are already using the same vendor ID, despite not being an Adafruit product.
Not sure how they picked their PIDs, as none of them seem registered.
Should I change just the PID to be something unique in the context of this repo?

I can try asking the folks at LilyGo. but, from what I've seen, they don't seem to be very responsive on Github or Twitter.

Reply:
> I used it because many other boards are already using the same vendor ID, despite not being an Adafruit product.
> Not sure how they picked their PIDs, as none of them seem registered.
> Should I change just the PID to be something unique in the context of this VID? like 0x80AD

We've allocated Adafruit PIDs for non-Adafruit devices when we're the ones adding support or at the request of a maker in our community.

We not doing this as much now because of the popularity of the S2. Companies should acquire their own VID. Open source hardware makers and open source software folks can get a PID from https://pid.codes (which I help administer.)

So, you should bug LilyGo but you can also request a pid.codes PID because you are unaffiliated with LilyGo.